sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/ContainerLock.java

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos;
+
+import com.azure.cosmos.implementation.guava25.base.Stopwatch;
```

Comment:
I figured it didn't matter since this was a test class and not exposed to anything outside of this project.  The apache-commons Stopwatch is much older than the guava implementation and doesn't return a Duration so interacting with it is much less elegant - that said, the usage is fairly limited so not really a problem, will get it updated.

Reply:
yeah, I agree, guava is much better, but unfortunately, it is not one of the approved dependencies for azure-sdk-for-java. For smaller usage, I guess apache-commons is fine, but if a time comes where we need it more, we can try to internally shade it.